safe/gis/expressions.py

```diff
@@ -39,7 +39,7 @@ def inasafe_impact_analysis_layer(field, feature, parent):
 
     index = layer.fieldNameIndex(field)
     if index < 0:
-        return None
+        return index
```

Comment:
This change for me is a blocker. Can we fix the function `inasafe_place_value_coefficient` to take if the value is None instead?

We are going to teach users to create their own template. Some of them will use this function without `inasafe_place_value_coefficient` and returning -1 looks like a bug from the user point of view. They need to write a QGIS expression like: ` if( inasafe_impact_analysis_layer('displaced') = -1, None, inasafe_impact_analysis_layer('displaced'))`